react/components/OrderInfo/OrderInfo.js

```diff
@@ -3,29 +3,46 @@ import PropTypes from 'prop-types'
 import parcelify from '@vtex/delivery-packages'
 import OrderHeader from './OrderHeader'
 import CustumerInfo from './CustumerInfo'
-import PaymentSummary from './Payment/PaymentSummary'
-import Shipping from './Shipping/Shipping'
+import PaymentSummary from '../Payment/PaymentSummary'
+import Shipping from '../Shipping/Shipping'
 import OrderTotal from './OrderTotal'
-import StorePickUp from './StorePickUp/StorePickUp'
+import OrderSummary from './OrderSummary'
+import StorePickUp from '../StorePickUp/StorePickUp'
+import OrderSplitNotice from './OrderSplitNotice'
 
 import { profileShape } from '../../proptypes/shapes'
 
 const OrderInfo = ({ data, profile }) => {
   const parcels = parcelify(data)
+  const delivery = parcels.filter((deliveryPackage) => deliveryPackage.deliveryChannel === 'delivery')
+  const pickup = parcels.filter((pickupPackage) => pickupPackage.deliveryChannel === 'pickup-in-point')
+  const takeaway = parcels.filter((takeawayPackage) => takeawayPackage.deliveryChannel === 'takeaway')
+  const multipleDeliveries = (delivery.length > 1)
   return (
     <div className="mv6 w-80-ns w-90 center">
+      <div className="bb b--muted-5">
+        <OrderSummary delivery={delivery} pickup={pickup} />
+      </div>
       <OrderHeader orderInfo={data} />
+      {
+        multipleDeliveries
+        && <OrderSplitNotice
+            deliveries={delivery.length}
+            pickups={pickup.length}
+            takeaways={takeaway.length}
+          />
+      }
       <div className="bb b--muted-5">
```

Comment:
Those repeated classes could be set as a variable to ease future updates.

Reply:
Or a component, like:

```js
<Block>
  <CustomerInfo profile={profile} />
</Block>
```

```js
// Block.js
import React from 'react'
import PropTypes from 'prop-types'

function Block({ children }) {
  return <div className="bb b--muted-5">{children}</div>
}

Block.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Block
```

Reply:
Created a new component called `OrderSection` to wrap substitute each of those div's.